src/css/profile/wearable/changeable/theme-circle/layout.less

```diff
@@ -133,6 +133,7 @@ body {
 		border: 0 none;
 		padding: 0;
 		text-indent: -9999px;
+		z-index: 5;
```

Comment:
Is there any particular reason why the z-index is 5?

Reply:
I just want to make it more higher than the ui-listview.
But I don't know why it makes the failure for the test.
Need to check the cause.